### DISCUSSION TITLE
Traduction in French

### DISCUSSION THREAD

**Discussion by Leaderfit** [mediacms-io/mediacms#426] [Q&A]
Hello,

Does the translation of the interface in French exist?
If so, how to set it up?

Thank you

**Comment by mgogoulos** [comment]
Hi, translations are not supported yet! Work has started but was never completed...

**Reply by Mozitux** [reply]
Eventually, if I have access to the work already started, I can finish the translation EN > FR

**Comment by mgogoulos** [comment]
They are part of the latest release, check it out on https://demo.mediacms.io/ 

The docs if you want to see how to improve it: 
https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#20-translations
